### DISCUSSION TITLE
Help Error 98

### DISCUSSION THREAD

**Discussion by socelillyhammer** [GerryDazoo/Slinger#604] [Q&A]
Hi all.  I had to reset my RPi and reconfig working settings. Now I screwed it up and hoping you can help - and i had to reget my pws from postman.

I keep getting this error code 98 and cannot figure out what is wrong.  My router settings are correct as well as port numbers.

thanks for the help

**_Server Error Code:_**

Version : 4.01 Running on Linux-6.1.21-v8+-aarch64-with-glibc2.31 pid= 4149 /home/antares/Slingbox/slingbox_server.py
Using config file config.ini
Connection Manager Running on port 51820 with 10 max streams using URL .
Building page for Slingbox
Reading Custom Remote definition from remote.txt
Starting Streamer Thread for Slingbox
Streamer Running:  10 config.ini SLINGBOX 51820 51820 8388608
starting up on  port 51820 8388608
51820 Checking for slingbox at ('', 5201) 0
 * Serving Flask app "slingbox_server" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
51820 Error connecting to  ('', 5201)
Broadcast Responder Running
ERROR While handling GETPORTNUMBERS request [Errno 98] Address already in use
Server IP address is 192.168.1.244
51820 No valid slingbox ip info found in config.ini
51820 Finding Slingboxes on local network. My IP Info =  192.168.1.244
51820 Found at 192.168.1.189 5219 " My Slingbox " FinderID 7F1FD40CF8655CAB6C ProductID 500
51820 Found at 192.168.1.182 5201 " Brooklyn Slingbox " FinderID 51D65ABF81456 ProductID 500
Found more than one slingbox on the local network.
    Please select the one you want to use and update the config file accordingly.
    
51820 Giving up. Sorry..
ERROR While handling GETPORTNUMBERS request [Errno 98] Address already in use

**### My CONFIG FILE**
[SLINGBOX]
sb1=bk
sb2=deal

[bk]
sbtype="350/500/M1/M2"
password=E1:aae59f04
;finderid=51D65ABF8145648462B36EEF22333070
ipaddress=192.168.1.182
port=5201
name=Brooklyn
Resolution=12
FrameRate=60
VideoBandwidth=2000
VideoSmoothness=59
IframeRate=5
AudioBitRate=32
VideoSource = 2
maxstreams=3

[deal]
sbtype="350/500/M1/M2"
password=
;finderid=7F1FD40CF8655CAB6C264592D4B32A15
ipaddress=192.168.1.189
port=5219
name=Deal
Resolution=12
FrameRate=60
VideoBandwidth=2000
VideoSmoothness=59
IframeRate=5
AudioBitRate=32
VideoSource = 2
maxstreams=3

[SERVER]
port=51820
enableremote=yes
URLbase=

[REMOTE]
code=2
Remote=remote.txt

**Comment by GerryDazoo** [comment]
You've made a hash of your config file. Please take a closer look at the unified-config sample file.
[SLINGBOX] should be [SLINGBOXES]
There should not be a [REMOTE] section
Error 98 in this case is probably caused by have two copies of the server running at the same time.
